src/backend/marsha/core/api.py

```diff
@@ -39,28 +39,26 @@ def update_state(request):
         The request on the API endpoint, it should contain a payload with the following fields:
```

Comment:
in your commit message:
s/to be refactor/to be refactored
s/fiels/fields
s/will contains/will contain
s/override/overriden
s/specific data to the related model/data specific to the related model